modules/income_limits/spec/requests/v1/limits_by_zip_code_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Income limits endpoints', type: :request do
+  describe '#get limitsByZipCode/:zip/:year/:dependents' do
+    it 'returns sample data' do
+      get '/income_limits/v1/limitsByZipCode/12345/2022/2'
+      expect(response).to have_http_status(:ok)
+      JSON.parse(response.body)
```

Comment:
**Question:** Is this checking that the response body exists as part of the tests? It's unclear, and it should try to check the data structure as well, such as the JSON having all the expected keys.

Reply:
I was holding off on adding those tests until we have connection to the oracle DB and can see the actual return data shape